project-resources/MEMBER_TRAVEL_FUND.md

```diff
@@ -206,13 +206,18 @@ The request from the
 [Cross Project council](https://github.com/openjs-foundation/cross-project-council)
 for a 2020 joint travel fund was approved in the amount of $60,000.
 
-
 ## 2021 OpenJS Foundation Board of Directors Allocation
 
 The request from the
 [Cross Project council](https://github.com/openjs-foundation/cross-project-council)
 for a 2021 joint travel fund was approved in the amount of $60,000. The line
-item in the budget was also renamed `community fund` as the expecation is that
+item in the budget was also renamed `community fund` as the expectation is that
 CPC will propose additional ways to spend this fund beyond just travel.
 
+## 2022 OpenJS Foundation Board of Directors Allocation
+
+The request from the
+[Cross Project council](https://github.com/openjs-foundation/cross-project-council)
+for a 2022 joint Community Fund was approved in the amount of $TBD.
```

Comment:
It would be nice to have an actual number here.

Reply:
Yes. Waiting on review from @mhdawson.